pkg/cache/context_test.go

```diff
@@ -2004,20 +1923,11 @@ func TestInitializeState(t *testing.T) {
 	assert.Equal(t, *pc.PreemptionPolicy, policy, "wrong preemption policy")
 	assert.Equal(t, pc.Annotations[constants.AnnotationAllowPreemption], constants.True, "wrong allow-preemption value")
 
-	// verify occupied / capacity on node
-	capacity, occupied, ok := context.schedulerCache.SnapshotResources(nodeName1)
-	assert.Assert(t, ok, "Unable to retrieve node resources")
-	expectedCapacity := common.ParseResource("4", "10G")
-	assert.Equal(t, expectedCapacity.Resources["vcore"].Value, capacity.Resources["vcore"].Value, "wrong capacity vcore")
-	assert.Equal(t, expectedCapacity.Resources["memory"].Value, capacity.Resources["memory"].Value, "wrong capacity memory")
-	expectedOccupied := common.ParseResource("1500m", "2G")
-	assert.Equal(t, expectedOccupied.Resources["vcore"].Value, occupied.Resources["vcore"].Value, "wrong occupied vcore")
-	assert.Equal(t, expectedOccupied.Resources["memory"].Value, occupied.Resources["memory"].Value, "wrong occupied memory")
-
```

Comment:
I re-added the assertions regarding "capacity". The occupied stuff is no longer calculated in the shim, so that's a goner.

Could you be a bit more specific about `foreignRunning` / `foreignPending`? I made no such update to the test.

Reply:
If we remove `foreignRunning` / `foreignPending`, which are created in lines 1856-1865, the test still pass.
Both pods are not added to cache and I think we should assert that behavior. Otherwise, we could remove both of them.

Reply:
Ah sorry. There are indeed `foreignRunning` `foreignPending` variables, I missed those. Let me update the test.

Reply:
@chenyulin0719 please check the changes now